cms/envs/test.py

```diff
@@ -192,6 +192,10 @@
 # No segment key
 CMS_SEGMENT_KEY = None
 
+# For most tests, setting JWT cookies during login is not necessary,
+# and raises errors when an oauth client has not been configured.
```

Comment:
Nit: "oauth client has not been configured" doesn't capture the issue that it requires seeding the test database, which is currently difficult to do.

Reply:
I added the following below in the code where this gets used:
```
# See https://github.com/edx/edx-platform/pull/19180 for discussion of alternative
#   solutions that did not work or were halted.
```
Do you want me to repeat that comment here?